packages/kbn-eslint-plugin-eslint/rules/no_restricted_paths.js

```diff
@@ -78,7 +78,7 @@ module.exports = {
                 },
                 allowSameFolder: { type: 'boolean' },
               },
-              additionalProperties: false,
+              additionalProperties: true,
```

Comment:
We can get rid of this right?

Reply:
That was to make `yarn lint` succeed, as it was failing.  I'll switch it back and see what we get.